README.md

```diff
@@ -1,25 +1,46 @@
-# goaci
+# proj2aci
 
-`goaci` is a simple command-line tool to build go projects into ACIs which confirm to the [app container specification][appc-spec].
+`proj2aci` is a command-line tool to build various projects into ACIs which conform to the [app container specification][appc-spec].
+
+Currently `proj2aci` supports rather simple cases:
+
+- Go projects buildable with `go get`
+- Rather simpler CMake projects
+
+`proj2aci` project is probably going to be superseded by [acbuild]
```

Comment:
This is basically telling people "don't use this" - is that really the message we want to send? Does it not provide any value?